grafana/resource_alerting_contact_point_notifiers.go

```diff
@@ -1257,7 +1258,7 @@ func (t threemaNotifier) schema() *schema.Resource {
 	}
 	r.Schema["recipient_id"] = &schema.Schema{
 		Type:        schema.TypeString,
-		Optional:    true,
+		Required:    true,
```

Comment:
Is this change related to the PR? It doesn't look like. Can we extract it to a PR of its own?

Reply:
Sure, I can split it out.

Quick note - I know this change appears to be compat breaking, but that schema was submitted in a PR just a couple days prior. None of these types have been released or published to the provider registry yet. So, users will never see a compat difference, we should not need to document anything special for this change.

Reply:
Also, it's _somewhat_ related to this PR. I added ~15 of these notifiers recently - it was one very large change, so I split them up into batches to make reviews easier. This change was simply an adjustment to a prior batch. It's not an unrelated field/resource, just a touch-up to the batch I had just submitted.

Given this, and given that it's not compat breaking per above, I feel OK with keeping it in the same PR.